src/test/scala/mesosphere/marathon/api/v2/PodsResourceTest.scala

```diff
@@ -619,7 +619,7 @@ class PodsResourceTest extends AkkaUnitTest with Mockito {
           implicit val podManager = PodManagerImpl(groupManager)
           val f = Fixture()
 
-          val response = f.podsResource.version("/id", "2008", f.auth.request)
+          val response = f.podsResource.version("/id", "2008-01-01T12:00:00Z", f.auth.request)
```

Comment:
is it OK that we must have changed this value in a test? Wasn't this a behavior that we were relying on also somewhere else?

Reply:
I don't think we can change the test here.

Reply:
see the comment below.